app/models/cart.rb

```diff
@@ -203,4 +203,9 @@ def total_price
       0.0
     end
   end
+
+  # may be replaced with paper-trail or similar at some point
```

Comment:
All I'm saying is that we shouldn't change what underlying value the `version` corresponds to, and `updated_at_i` was a way to state that more explicitly. I think we're on the same page?